lib/handler/pool/worker.ex

```diff
@@ -0,0 +1,27 @@
+# This is a generic GenServer (ie GenGenServer 🤣)
```

Comment:
This line doesn't really make sense any more since the module name is no longer `GenGenServer`.